api/src/pcapi/scripts/external_users/populate_beamer_pro_users.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import math
+import typing
+
+import sqlalchemy.orm as sqla_orm
+
+from pcapi.connectors import beamer
+import pcapi.core.external.attributes.api as attributes_api
+import pcapi.core.external.attributes.models as attributes_models
+import pcapi.core.users.models as users_models
+
+
+CHUNK_SIZE = 500
+
+logger = logging.getLogger(__name__)
+
+
+def populate_beamer_pro_users(from_user_id: typing.Optional[int] = None) -> None:
+    pro_user_attributes = _get_pro_user_attributes(from_user_id)
+
+    for pro_attribute in pro_user_attributes:
```

Comment:
uh, c'est dur le vendredi, j'ai cru qu'on bouclait sur les attributs et qu'on les mettait à jour un par un. 

J'aurais bien vu un pro_with_attributes in pros_with_attributes: mais c'est du détail !